### ISSUE TITLE
no simple features geometry column present

### ISSUE THREAD

**Issue by Tazinho** [edzer/sfr#356]
Have there been any significant changes in the last month
I am sure that this code worked before:

```r
> st_sf(points = st_sfc(st_point(c(0, 0)),
+ st_point(c(0, 1)),
+ crs = 4326))
Error in st_sf(points = st_sfc(st_point(c(0, 0)), st_point(c(0, 1)), crs = 4326)) : 
  no simple features geometry column present
```

**Comment by edzer** [comment]
This was solved in #348

**Comment by Tazinho** [comment]
Thanks!
